test/functional/api/v2/template_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_plugin_helper'
+
+class Api::V2::TemplateControllerTest < ActionController::TestCase
+
+  test "should import" do
+    post :import, { 'repo':'/does-not-exist' }
+    response = ActiveSupport::JSON.decode(@response.body)
+    assert_equal response['error']['message'], "Using file-based synchronization, but couldn't find /does-not-exist"
+    assert_response :error
+  end
```

Comment:
I done it this way because if we send it a git path its takes a fair amount of time to clone & import also there's potential for unrelated failures? 

But I'm happy to add a git URL as well..